### PR TITLE
created 5 pages with images and text

### PR DESCRIPTION
**Pull request by JimChengINF7420SS19**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,55 @@
...
+        <li><a href="about us.html">About us</a></li>
+        <li><a href="specialist area.html">Specialist Area</a></li>
+        <li><a href="claims.html">Claims</a></li>
+        <li><a href="contact us.html">Contact us</a></li>
+       </ul>
+    </nav>
+  </header>
+  <main>
+    <div>
+      <img src="" alt="index picture"> </div>
```
Comment (axamei) [inline index.html]: missing src. use a placeholder if you do not have this image yet.

File: index.html [hunk 2]
```diff
@@ -4,10 +4,55 @@
...
+      </div>
+<h3>Specialist Area</h3>
+<div>
+  <ul>
+      <li><a href="home.html">Home</a></li>
+      <li><a href="travel.html">Travel</a></li>
+      <li> <a href="car.html">Car</a></li>
+      <li><a href="personal accident.html">Personal Accident</a></li>
+  </ul>
+  </main>
```
Comment (axamei) [inline index.html:48]: missing closing divs.

File: index.html [hunk 3]
```diff
@@ -4,10 +4,55 @@
...
+  <footer>
+<div>
+  <img src="" alt="Professional Organization Logo"></div>
```
Comment (axamei) [inline index.html]: missing src. use a placeholder if you do not have the image yet.

File: index.html [hunk 4]
```diff
@@ -4,10 +4,55 @@
...
 </head>
 <body>
-  <h1>Hello World!</h1>
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/_a0GdDydowc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <header>
+
+    <nav>
+
+      <img src="images/yourlogohere.png" alt=" company logo ONE">
+      <img src="images/company name.jpg" alt=" company name TWO">
```
Comment (axamei) [inline index.html]: try to avoid spaces in filenames. rename the file and edit the src.

File: index.html [hunk 5]
```diff
@@ -4,10 +4,55 @@
...
+       <ul>
+        <li> <a href="home.html">Home</a></li>
+        <li><a href="about us.html">About us</a></li>
+        <li><a href="specialist area.html">Specialist Area</a></li>
+        <li><a href="claims.html">Claims</a></li>
+        <li><a href="contact us.html">Contact us</a></li>
```
Comment (axamei) [inline index.html]: try to avoid spaces. use dashes instead. change file to contact-us.html and edit this href.

File: index.html [hunk 6]
```diff
@@ -4,10 +4,55 @@
...
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/_a0GdDydowc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <header>
+
+    <nav>
+
+      <img src="images/yourlogohere.png" alt=" company logo ONE">
+      <img src="images/company name.jpg" alt=" company name TWO">
+       <ul>
+        <li> <a href="home.html">Home</a></li>
+        <li><a href="about us.html">About us</a></li>
```
Comment (axamei) [inline index.html]: try to avoid spaces. use dashes instead. change file to about-us.html and edit this href.

File: index.html [hunk 7]
```diff
@@ -4,10 +4,55 @@
...
+        <li><a href="specialist area.html">Specialist Area</a></li>
```
Comment (axamei) [inline index.html]: try to avoid spaces. use dashes instead. change file to specialist-area.html and edit this href.

File: index.html [hunk 8]
```diff
@@ -4,10 +4,55 @@
...
+      <h2>Specialist Area</h2>
+      <div>
+        <ul>
+            <li><a href="home.html">Home</a></li>
+            <li><a href="travel.html">Travel</a></li>
+            <li> <a href="car.html">Car</a></li>
+            <li><a href="personal accident.html">Personal Accident</a></li>
+        </ul>
+      </div>
+<h3>Specialist Area</h3>
```
Comment (axamei) [inline index.html]: is this a duplicate of what is above?

File: about us.html [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+  <title>Jim Cheng - Portfolio</title>
+</head>
+<body>
+  <header>
+
+    <nav>
+
+
+        <img src="images/yourlogohere.png" alt=" company logo ONE">
+        <img src="images/company name.jpg" alt=" company name TWO">
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+     
+       <ul>
+        <li> <a href="home.html">Home</a></li>
+        <li><a href="about us.html">About us</a></li>
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+        <li><a href="specialist area.html">Specialist Area</a></li>
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+        <li><a href="claims.html">Claims</a></li>
+        <li><a href="contact us.html">Contact us</a></li>
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+  <main>
+    <div>
+        <h3>
+            <p>Home Page Marketing Message</p>
+        </h3>
+      <div>
+     <div>
+         <h2>About us</h2>
+            <ul>
+                <li> <a href="CTK & Co.html">Why CTK & Co.</a></li>
```
Comment (axamei) [inline about us.html]: avoid using special characters like `&` in file names. rename.

File: about us.html [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+                <li><a href="our team.html">Our Team</a></li>
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+                <li><a href="our partners.html">Our Partners</a></li>
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sed velit dignissim sodales ut eu sem integer. Id diam maecenas ultricies mi eget mauris pharetra et. Tincidunt eget nullam non nisi est sit. Sit amet nisl suscipit adipiscing bibendum est ultricies integer quis. Mauris in aliquam sem fringilla ut morbi tincidunt augue. Est velit egestas dui id ornare arcu odio ut. Quis hendrerit dolor magna eget est lorem. Varius duis at consectetur lorem donec massa. Curabitur gravida arcu ac tortor dignissim convallis. At auctor urna nunc id cursus metus aliquam. Odio ut enim blandit volutpat maecenas. Lobortis mattis aliquam faucibus purus in massa tempor. Odio euismod lacinia at quis risus sed vulputate odio ut. Justo nec ultrices dui sapien eget mi proin sed libero. Nunc mi ipsum faucibus vitae aliquet nec ullamcorper. Mi sit amet mauris commodo quis imperdiet massa tincidunt. Ut porttitor leo a diam sollicitudin. Faucibus turpis in eu mi. Amet nulla facilisi morbi tempus iaculis urna id. </p>
+            <img src="images/about us dummy-454x280-SkiLift.jpg" alt=" About us picture">
```
Comment (axamei) [inline about us.html]: avoid spaces.

File: about us.html [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+      <div>
+        <div>
+            <ul>
+                <li> <a href="CTK & Co.html">Why CTK & Co.</a></li>
```
Comment (axamei) [inline about us.html]: is this a duplicate of the set of links above?

File: about us.html [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+                <li><a href="our team.html">Our Team</a></li>
+                <li><a href="our partners.html">Our Partners</a></li>
+            </ul>
+        </div>
+        
+  </main>
+  <footer>
+<div>
+  <img src="" alt="Professional Organization Logo"></div>
```
Comment (axamei) [inline about us.html]: missing src. use placeholder for now.

File: about us.html [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+            <img src="images/about us dummy-454x280-SkiLift.jpg" alt=" About us picture">
+      <div>
+        <div>
+            <ul>
+                <li> <a href="CTK & Co.html">Why CTK & Co.</a></li>
+                <li><a href="our team.html">Our Team</a></li>
+                <li><a href="our partners.html">Our Partners</a></li>
+            </ul>
+        </div>
+
```
Comment (axamei) [inline about us.html]: missing closing divs.

File: claims.html [hunk 20]
```diff
@@ -0,0 +1,53 @@
...
+  <header>
+
+    <nav>
+
+            <img src="images/yourlogohere.png" alt=" company logo ONE">
+            <img src="images/company name.jpg" alt=" company name TWO">
+    
+       <ul>
+        <li> <a href="home.html">Home</a></li>
+        <li><a href="about us.html">About us</a></li>
```
Comment (axamei) [inline claims.html]: avoid spaces.

File: claims.html [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+        <li><a href="specialist area.html">Specialist Area</a></li>
```
Comment (axamei) [inline claims.html]: avoid spaces.

File: claims.html [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+        <li><a href="claims.html">Claims</a></li>
+        <li><a href="contact us.html">Contact us</a></li>
```
Comment (axamei) [inline claims.html]: avoid spaces.

File: claims.html [hunk 23]
```diff
@@ -0,0 +1,53 @@
...
+       </ul>
+    </nav>
+  </header>
+  <main>
+    <div>
+      <img src="images/claims dummy-454x280-Dugs.jpg" alt="image in Claims"> </div>
```
Comment (axamei) [inline claims.html]: avoid spaces.

File: claims.html [hunk 24]
```diff
@@ -0,0 +1,53 @@
...
+      <p>text for Claims :Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Comment (axamei) [inline claims.html]: missing closing p

File: claims.html [hunk 25]
```diff
@@ -0,0 +1,53 @@
...
+    
+        <p><Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum./p>
```
Comment (axamei) [inline claims.html]: text should be in between tags not inside them.

File: claims.html [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+        <p> We have specialist experience in car, home, personal accident and travel insurance. In each of our practice areas, we strive to find innovative and bespoke solutios for every risk we place. </p>
+    
+        <p>
+            John Doe
+            Telephone: 415 386 6449
+            john@ctkco.com
+        </p>
+
+    <div>
+
```
Comment (axamei) [inline claims.html]: missing closing div.

File: claims.html [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+  </main>
+  <footer>
+<div>
+  <img src="" alt="Professional Organization Logo"></div>
```
Comment (axamei) [inline claims.html]: missing src. use placeholder for now.

File: contact us.html [hunk 28]
```diff
@@ -0,0 +1,55 @@
...
+        <h1>Contact us </h1>
+          <p> We look forward to speaking with you. 
+
+           <p>  Tel: 415 386 6444  </p>   
+           <p>  Fax: 415 386 5444 </p>   
+           <p> Email for general inquiries: info@ctkco.com </p> 
+            
+            
+           <p>   Head office:   </p> 
+            <p>   4500 Cass Street</p>
```
Comment (axamei) [inline contact us.html]: you can use the `<address>` tag for addresses.

**Review by axamei — CHANGES_REQUESTED** [review]
There are some errors in your code. I will add a commit to show you how to resolve some of them. You will have to fetch and pull this html branch using GitHub Desktop. You can then make changes, add commits, and then push your changes.

File: about us.html (hunk 9 above)
Comment (JimChengINF7420SS19) [reply about us.html]: Thanks. I did the changed in VSC.

**Pull request merged**